MAKE/Makefile.carrington_gcc_openmpi

```diff
@@ -11,20 +11,21 @@
 # module load gnu9/9.3.0
 # module load openmpi4/4.0.5
 # module load pmix/3.1.4
-
+# Oneliner for command line:
+# module purge ; module load gnu9/9.3.0 ; module load openmpi4/4.0.5 ; module load pmix/3.1.4
```

Comment:
Add: 
remember to compile inside interactive session:
/usr/bin/srun -M carrington --interactive -n 16 --mem=4G --time=00:20:00 -p short --pty bash

Reply:
Actually there the "old" interactive call still works. But good reminder. Shall I instead add "remember to compile in an interactive session (see wiki for current best practices)"?

Reply:
sounds good (and make sure the wiki is up to date :) )

Reply:
Juha confirmed that interactive is deprecated also on Carrington.